src/main/java/pl/hk/zadanie_26/part/PartService.java

```diff
@@ -0,0 +1,63 @@
+package pl.hk.zadanie_26.part;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class PartService {
+    private final PartRepository partRepository;
+
+    public PartService(PartRepository partRepository) {
+        this.partRepository = partRepository;
+    }
+
+    public List<Part> productCatalog() {
+        return partRepository.findAll();
+    }
+
+    public void addPart(Part part) {
+        partRepository.save(part);
+    }
+
+    public Part findByIdPart(Long id) {
+        Optional<Part> part = partRepository.findById(id);
+        if (part.isPresent()) {
+            return part.get();
+        } else {
+            throw new RuntimeException();
+        }
+    }
+
+    public void editPart(Part part) {
+        Part part1 = findByIdPart(part.getId());
+        part1.setName(part.getName());
+        part1.setPrice(part.getPrice());
+        part1.setCategory(part.getCategory());
+        partRepository.save(part1);
+    }
+
+    public List<Part> printListOfProductsInCart() {
+        return partRepository.findAllBySelected(true);
+    }
+
+    public void addProductToCart(Long id) {
+        Part part = findByIdPart(id);
+        part.setSelected(true);
```

Comment:
Koszyk będzie działał tylko dla jednego użytkownika. Jak będzie więcej osób korzystać z tej aplikacji, to jak jedna osoba doda/usunie z koszyka, to wszystkim się będzie ten koszyk zmieniał. Tutaj raczej należałoby dodać logikę mówiącą jaki użytkownik dodał to sobie do koszyka